index.js

```diff
@@ -3,134 +3,183 @@ const countries = require('./countries');
+        choices: [
+          {
+            value: true,
+            showFields: [
+              'originLatitude', 'originLongitude'
+            ]
+          }
+        ]
+      },
+      {
+        name: 'originLatitude',
+        label: 'Origin Latitude',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'originLongitude',
+        label: 'Origin Longitude',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'bounds',
+        label: 'Bounds',
+        type: 'boolean',
+        htmlHelp: 'The area in which to search for places. <a href="https://developers.google.com/maps/documentation/javascript/places-autocomplete#add_autocomplete" target="_blank">More information here</a>',
+        choices: [
+          {
+            value: true,
+            showFields: [
+              'strictBounds', 'northEastLatitude', 'northEastLongitude', 'southWestLatitude', 'southWestLongitude'
+            ]
+          }
+        ]
+      },
+      {
+        name: 'northEastLatitude',
+        label: 'North East Latitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'northEastLongitude',
+        label: 'North East Longitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'southWestLatitude',
+        label: 'South West Latitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'southWestLongitude',
+        label: 'South West Longitude Bound',
+        type: 'float',
+        required: true
+      },
+      {
+        name: 'strictBounds',
+        label: 'Strict Bounds',
+        type: 'boolean',
+        htmlHelp: 'Returns only those places that are strictly within the region defined by the given bounds. <a href="https://developers.google.com/maps/documentation/javascript/places-autocomplete#add_autocomplete" target="_blank">More information here</a>'
+      },
+      {
+        name: 'countries',
+        label: 'Countries',
+        type: 'array',
+        titleField: 'country',
+        limit: 5,
+        help: 'Restrict search to one or multiple countries (5 at max due to Google restriction). Might be not compatible with origin or bounds values.',
+        schema: [
+          {
+            name: 'country',
+            label: 'Country',
+            type: 'select',
+            choices: countries
+          }
+        ]
+      }, {
+        name: 'splitAddress',
+        label: 'Split Address',
+        type: 'boolean',
+        choices: [
+          {
+            value: true,
+            showFields: ['addressParts', 'displaySplitAddress']
+          }
+        ]
+      }, {
+        name: 'addressParts',
```

Comment:
Back to `checkboxes` with default settings. If someone wants to change the order of appearance of the broken out fields, they can just override the template I guess.